### ISSUE TITLE
multiple users in one tread

### ISSUE THREAD

**Issue by Jarifa** [Dragory/modmailbot#311]
Would it be possible to have the possibility to open a new thread including multiple members?
Right now I'm using the bot for a small private tournament and we use the bot to document important communication and sometimes we like to have a discussion with multiple people involved. would it be possible in the future to add multiple people to a thread?

I love your work <3
Thanks!

**Comment by smooreace** [comment]
Thats called "Group Chat"

**Comment by Jarifa** [comment]
And how would I have logs with people deleting stuff in "group chat"?
Don't be a smartass. I'm sure you can use the few brain cells to understand what I mean.

**Comment by smooreace** [comment]
Its called... PERMISSIONS dumbass! Use some of those brain cells!

**Comment by sporkwitch** [comment]
@Jarifa not productive. You're here asking for enhancements and help; that's not the way to get it.

@smooreace unless it's changed since I last used it, group chats don't have perms, they're like any other DM. Either way, it wouldn't provide the easy documentation of edits and deletions he's looking for (even if you can't delete others, discord doesn't allow you to 0revent people editing or deleting their own messages).

**Comment by smooreace** [comment]
True... but a channel with selective access can be used the same way with
any of a dozen bots mirroring input to a "recording" channel. The group
chat suggestion was made before the lack of integrity of chat contributors
was mentioned.

**Comment by sporkwitch** [comment]
Seems rather convoluted.  The bigger issue with implementing the request is that Discord group chats only allow Friends to invite people, so I'm not sure if it's even possible (can bot accounts even be friends with people?)

**Comment by Jarifa** [comment]
I am not sure how you even think a group chat is even comparable with a multiple-user thread.
by using that logic, why even use this bot at all and just do everything through private messages? 
Like @sporkwitch mentioned, its about the documentation, and more importantly, centralization of discussion.

**Comment by smooreace** [comment]
It sounds like that's a pretty valid point. Why ARE you trying to use a
support ticket process to manage group conversations? Since I/we have no
clue what you're actually trying to accomplish in the grand scheme of
things then all I can do is offer logical solutions based on the limited
data I have. Stop bitching about the assistance offered and make yourself
more clear! What you originally asked for is easily done in a private text
channel.

"sometimes we like to have a discussion with multiple people involved" nuff
said.

**Comment by Jarifa** [comment]
I literally explained this in the message before you. I thought what I meant in the original post was straightforward. Why would I otherwise ask for this feature if its possible in vanilla discord? You could have simply asked me or said to me that you didn't understand but instead you "contributed" by making a passive-agressive remark. and then stomp on me for responding to it in an annoyed manner. 

I explained in messages after what I meant and I don't get why you are still responding with "What you originally asked for is easily done in a private text".
Yes, maybe I was a bit unclear in the original message but I clarified so please can you stop trying to push the fact that you want to be right.

**Comment by smooreace** [comment]
Someone needs a Midol!!

**Comment by Jarifa** [comment]
I'm pretty sure I've explained my points pretty well right now so please refrain from posting if you have nothing to add. It just makes you look stupid.

**Comment by sporkwitch** [comment]
Both of you need to grow up. No one's going to help you when you're bickering like 5 year olds.

**Comment by Dragory** [comment]
I marked this as `enhancement`, but it's unlikely this will be implemented in the near future (if at all). The bot is mainly designed for 1-to-many discussions (1 user, multiple mods), like a ticket system, and as such, changing it to support multiple people on both ends would require a lot of refactoring.
